hazelcast/src/main/java/com/hazelcast/config/CacheConfig.java

```diff
@@ -87,6 +90,7 @@
      */
     private boolean disablePerEntryInvalidationEvents;
 
+    private TenantControl tenantControl = NOOP_TENANT_CONTROL;
```

Comment:
should we add `tenantControl` to `toString`, `hashCode`, `equals` methods?

Reply:
`tenantControl` field is only meant to be populated and used internally at runtime, so it shouldn't participate in equality check. I pushed a commit to turn `get/setTenantControl` methods package private so they are not accessible to users.